### DISCUSSION TITLE
OSError: [Errno 95] Operation not supported: '/content/drive/dreambooth_lora_cmd.yaml

### DISCUSSION THREAD

**Discussion by AizoLau** [Linaqruf/kohya-trainer#183] [Q&A]
Kohya LoRA Dreambooth（oldversion) 
I run step5.3 and error occurred. How can I fix it

**Comment by Linaqruf** [comment]
I think using new version can fix it.

**Reply by AizoLau** [reply]
But new version is too complicated to use for me. Is there any tutorials how to use it? like this -[LoRA training guide](https://www.reddit.com/r/StableDiffusion/comments/11vw5k3/lora_training_guide_version_3_i_go_more_indepth/)
By the way, this tutorials helps me fix the error

**Reply by Linaqruf** [reply]
pretty sure old version is complicated than new version, 
old version need to be updating xformers, triton, and it has many bugs comparing to the newer version, and you dont need to that in new version

**Reply by Linaqruf** [reply]
this dont exist in newer version, 
![image](https://user-images.githubusercontent.com/50163983/231332686-a4456926-fcbc-41d3-9d18-720860ee44eb.png)

this pruned to one cell in newer version
![image](https://user-images.githubusercontent.com/50163983/231332737-c44ce4e4-c394-4c2c-8c95-f86ea7d9fefa.png)

you dont need to do this anymore in newer version
![image](https://user-images.githubusercontent.com/50163983/231332946-c8c04dbb-03f7-4ac7-95de-4db81138e40b.png)

this cell simplified in newer version, it only has 2 field
![image](https://user-images.githubusercontent.com/50163983/231332909-aa674ff9-415f-4b8d-9f7e-c6de87aa0f39.png)

this cell also simplified in newer version
![image](https://user-images.githubusercontent.com/50163983/231333137-b2e6326c-559a-450f-87a8-ba767556e879.png)

this also dont exist in newer version
![image](https://user-images.githubusercontent.com/50163983/231333255-6ef1cca3-696e-4c54-aaf3-00d9c251fa0b.png)


the only new celll i add was dataset config.

**Reply by AizoLau** [reply]
does the new version still need to provide trigger word or otehr descriptions in a .txt file by myself?
